examples/operators/subtract/run.py

```diff
@@ -30,9 +33,10 @@ def subtract(
 
     assert stop_spec - start_spec == stop_bg - start_bg
 
-    output_data = {
-        "eloss": eloss[start_spec:stop_spec],
-        "spec": spec[start_spec:stop_spec] - background[start_bg:stop_bg],
+    outputs: Dict[PortKey, RawPort] = {
```

Comment:
Why do we need inline type annotations?

Reply:
We don't strictly, but I usually write the code with and empty dict and explicit types, so that once I start filling in the contents of the dict I get notified by the editor if I made any mistake.